src/js/form-builder.js

```diff
@@ -1609,8 +1609,7 @@ function FormBuilder(opts, element, $) {
 
     const currentId = currentItem.attr('id')
     const type = currentItem.attr('type')
-    const ts = new Date().getTime()
```

Comment:
in the future there is also the possibility to use `performance.now` in the `nameAttr` function for higher resolution time stamps

Reply:
Yes. We really should be using `Date.now()` instead of `new Date().getTime()` to remove the object creation. We can also replace that call with `performance.timeOrigin + performance.now()`